### DISCUSSION TITLE
Where is the dockerfile file of the dashboard? How Do I Compile the Dashboard Using DockerFile?

### DISCUSSION THREAD

**Discussion by m2a2172** [kubernetes/dashboard#8264] [Q&A]
Where is the dockerfile file of the dashboard? I can two files, aio/Dockerfile, aio/develop/Dockerfile .
which one ?  How Do I Compile the Dashboard Using DockerFile?

**Comment by floreks** [comment]
https://github.com/kubernetes/dashboard/blob/master/modules/api/Dockerfile
https://github.com/kubernetes/dashboard/blob/master/modules/web/Dockerfile

Every module has its own Dockerfile now, but it can't be easily used on its own since it relies on our build pipeline and requires the application to be built first.
